packages/@okta/vuepress-site/docs/guides/device-context/main/index.md

```diff
@@ -0,0 +1,151 @@
+* Network: Are they signing in from a safe network?
+* Travel: Are they signing in from a new location?
+* What groups does the user belong to?
+* Are they signing in through SSO?
+
+For example, a user signing in to your website from a new device or new country may signal a risky situation. Requiring additional authenticators&mdash;email, phone SMS, or biometrics&mdash;gives extra identity assurance in these circumstances.
+
+In this example, you'll implement a simple AMFA policy requiring two authentication factors from new devices and only one factor from known devices. The following diagram illustrates this flow:
+
+<div class="full">
+
+![Flow diagram showing how adaptive multifactor authentication works for new devices](/img/advanced-use-cases/device-context-adaptive-mfa-flow.png)
+
+</div>
+
+Follow these steps to enable this AMFA policy in your app:
+
+1. Update your application to send the `X-Device-Token` header to your Org.
+2. Update or create a new global session policy to require a second authentication factor for sign-in requests from new devices.
+3. Update or create an authentication policy for your app to require and identify a second authentication factor for new devices.
+
+> **Note**: For testing purposes, we suggest creating new policies rather than adding AMFA rules to your existing policies.
+
+### Create a new global session policy for AMFA
+
+1. Open the **Admin Console** for your org.
+2. Choose **Security** > **Global Session Policy** to show the available global session policies.
+3. Click **Add Policy**.
+4. Set a **Policy Name**; for example, "New Device AMFA Policy".
+5. Set **Assign to Groups** to your target user groups; for example, "Test Users".
+6. Click **Create Policy and Add Rule** to save the new policy.
+
+After creating the policy, the **Add Rule** dialog appears.
+
+1. Set a **Rule Name**; for example, "New Device AMFA Rule".
+2. Set **Behavior is** to **New Device**. This is where the `X-Device-Token` header is checked.
+3. Set **Establish the user session with** to **A password**.
+4. Set **Multifactor authentication (MFA) is** to **Required**.
+5. Set **Establish the user session with** to **A password**.
+6. Verify that **Users will be prompted for MFA** is set to **At every sign-in**.
+7. Click **Create Rule** to create the rule.
+
+These rules declare that:
+
+* Every user starts the sign-in process with a password
+* If "New Device" is true, a second authentication factor is required, and users will be prompted as such for each sign-in process.
+
+> **Note**: See [Add a global session policy rule](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/add-okta-sign-on-policy-rule.htm) to learn how the different MFA settings interact.
+
+### Create a new app authentication policy for AMFA
+
+Create a new authentication policy for your app that contains two rules that state:
+
+1. By default, users authenticate with a username and password only.
```

Comment:
I'm pretty sure this list should be unordered. Users aren't required to do these steps in order.
```suggestion
- By default, users authenticate with a username and password only.
```

Reply:
@brentschaus-okta I'm not changing this to an unordered list. It's a numbered list to identify the two rules per se and not an order to them. They aren't steps - just what the rules are.